src/Caching/StackExchangeRedis/src/RedisCache.cs

```diff
@@ -49,8 +51,8 @@ public class RedisCache : IDistributedCache, IDisposable
     private const long NotPresent = -1;
     private static readonly Version ServerVersionWithExtendedSetCommand = new Version(4, 0, 0);
 
-    private volatile IConnectionMultiplexer _connection;
-    private IDatabase _cache;
+    private volatile IConnectionMultiplexer? _connection;
```

Comment:
Why are these nullable? It seems like they should be required based on usage in the code.

Reply:
Members are considered nullable unless they are initialized as part of object serialization (constructor / field or property initializer). These fields are lazy initialized (which rely on a null check), hence nullable.